log_test.go

```diff
@@ -82,9 +82,9 @@ func TestNewLog(t *testing.T) {
 		l.Log("onefish")
 		l.SetPrefix("[PREFIX]")
 		l.Log("twofish")
-		l.SetPrefix("%{color:red}[COLOR PREFIX]")
```

Comment:
So I thought this was a real test of a real feature, i.e that color works inside prefix.  Why does it have to be removed, does it fail on Travis or something?